crates/storage/src/lib.rs

```diff
@@ -35,6 +37,10 @@ table!(
     /// Account codes table.
     ( AccountCodes ) AccountCodeHashRLP => AccountCodeRLP
 );
+table!(
+    /// Receipts table.
+    ( Receipts ) BlockNumber => ReceiptRLP
```

Comment:
There are multiple receipts per block, so we want this to be a `dupsort` table, with `BlockNumber` as key and `u64` as subkey.